entanglement_forging/core/wrappers/entanglement_forged_vqe_result.py

```diff
@@ -195,3 +207,18 @@ def energy_std_each_parameter_set(self) -> Optional[List[Union[float, int]]]:
     def eval_count(self) -> Optional[int]:
         """Returns evaluation count."""
         return self._eval_count
+
+    @property
+    def num_particles(self) -> Optional[Tuple[complex, complex]]:
+        """Returns number of particles."""
+        return self._eval_count
```

Comment:
I think you may have forgotten to update this return

Reply:
Yes! that's what was causing that crazy 401. good catch